src/index.css

```diff
@@ -11,3 +13,157 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.header {
+  text-align: center;
+}
+
+.title {
+  background: linear-gradient(to right, violet, orange, green, cyan, blue);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-family: "Young Serif", serif;
+  font-size: 50px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.question {
+  color: rgb(79, 79, 79);
+}
+.inputContainer {
+  display: flex;
+  align-items: left;
+  flex-direction: column;
+  text-align: left;
+  background: rgb(232, 231, 231);
+  border: 1px solid;
+  border-color: black;
+  padding: 10px 0 30px 30px;
+  margin-bottom: 20px;
+  width: 800px;
+  box-shadow: 11px 7px 0px 2px #000000;
+}
+
+.input {
+  padding: 10px;
+  width: 90%;
+}
+
+.textBox {
+  border: 1px solid;
+  padding: 20px;
+  margin: 20px 0 20px 0;
+  width: 750px;
+  box-shadow: 11px 7px 0px 2px #000000;
+}
+
+.likeBtn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+.like-time {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+.heart {
+  border: none;
+  font-size: 30px;
+  border-radius: 50%;
+  padding: 8px 10px 8px 10px;
+  background: rgb(222, 222, 222);
+  animation: pulse 2.5s infinite;
+  scale: 90%;
+  cursor: pointer;
+}
+
+@keyframes pulse {
+  0% {
+    box-shadow: 0 0 0 0 rgb(222, 222, 222);
+  }
+
+  70% {
+    box-shadow: 0 0 0 0.2rem rgb(222, 222, 222);
+  }
+
+  100% {
+    box-shadow: 0 0 0 0 rgb(222, 222, 222);
+  }
+}
+
+.time {
+  font-size: 18px;
+  color: gray;
+  margin-left: 200px;
+}
```

Comment:
This works really well with screens larger than 665px, but not the smaller ones. The time falls out of the box. Maybe you can remove the margin-left here and work only with styling of the parent element (sorry, am on mobilephone and cannot test)